src/main/java/ua/vedroid/dao3/dao/jdbc/CarDaoJdbcImpl.java

```diff
@@ -0,0 +1,206 @@
+package ua.vedroid.dao3.dao.jdbc;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import ua.vedroid.dao3.dao.CarDao;
+import ua.vedroid.dao3.exceptions.DataProcessingException;
+import ua.vedroid.dao3.lib.Dao;
+import ua.vedroid.dao3.model.Car;
+import ua.vedroid.dao3.model.Driver;
+import ua.vedroid.dao3.model.Manufacturer;
+import ua.vedroid.dao3.util.ConnectionUtil;
+
+@Dao
+public class CarDaoJdbcImpl implements CarDao {
+    @Override
+    public Car create(Car car) {
+        String query = "INSERT INTO cars(m_id, model) VALUES (?, ?);";
```

Comment:
I don't know if you're violating checkstyle like that... 
![image](https://user-images.githubusercontent.com/65593125/104955380-da92f380-59d2-11eb-96e6-0dedf81cd126.png)
`m` is good for shortened variant but as a full name I could be a little bit frustrated about `m` meaning.

Reply:
agree